transports/quic/src/transport.rs

```diff
@@ -574,6 +617,35 @@ impl<P: Provider> Listener<P> {
             }
         }
     }
+
+    fn update(
+        &mut self,
+        is_webtransport: bool,
+        cert_hashes: &Vec<Multihash<64>>,
+        noise_cfg: libp2p_noise::Config,
```

Comment:
```suggestion
        webtransport_cfg: Option<libp2p_noise::Config>,
        cert_hashes: &Vec<Multihash<64>>,
```
No need for noise config if it's not webtransport.